feature-libs/organization/administration/core/store/actions/org-unit.action.ts

```diff
@@ -145,6 +159,15 @@ export class LoadOrgUnitNodes extends StateUtils.EntityLoadAction {
 export class LoadOrgUnitNodesFail extends StateUtils.EntityFailAction {
   readonly type = LOAD_UNIT_NODES_FAIL;
 
+  /**
+   * @deprecated Please use `error` parameter other than `null` or `undefined`.
+   *
+   *             Note: Allowing for `null` or `undefined` will be removed in future versions
+   *             together with the feature toggle `ssrStrictErrorHandlingForHttpAndNgrx`.
+   **/
+  constructor(payload: { error: null | undefined });
+  // eslint-disable-next-line @typescript-eslint/unified-signatures
+  constructor(payload: { error: ActionErrorProperty });
   constructor(public payload: any) {
     super(ORG_UNIT_NODE_LIST, ORG_UNIT_NODES, payload.error);
```

Comment:
Note: exceptional case: constructor param was `payload: any`, but looking up a nested property `payload.error` inside.
Usually (in other places) if `payload: any`, then the payload was itself an error.